packages/editor/src/components/widgets/table/row/SelectRow.tsx

```diff
@@ -2,8 +2,20 @@ import './SelectRow.css';
 import { Row } from '@tanstack/react-table';
 import { ReactNode } from 'react';
 
-export const SelectRow = <TData extends object>({ row, children }: { row: Row<TData>; children: ReactNode }) => (
-  <tr className='selectable-row' data-state={row.getIsSelected() ? 'selected' : ''} onClick={row.getToggleSelectedHandler()}>
+export const SelectRow = <TData extends object>({
+  row,
+  children,
+  isNotSelectable
+}: {
+  row: Row<TData>;
+  children: ReactNode;
+  isNotSelectable?: boolean;
+}) => (
```

Comment:
as soon as it breaks all attributes below each other I start move the property type definition outside. Just a personal preference but I think it is much more readable:

```ts
type SelectRowProps<TData> = {
  row: Row<TData>;
  children: ReactNode;
  isNotSelectable?: boolean;
}

export const SelectRow = <TData extends object>({ row, children, isNotSelectable }: SelectRowProps<TData>) => (
```